external/mcpclient/package.json

```diff
@@ -30,7 +30,8 @@
   },
   "peerDependencies": {
     "@modelcontextprotocol/sdk": "^1.9.0",
-    "@microsoft/teams.ai": "*"
+    "@microsoft/teams.ai": "*",
+    "@microsoft/teams.common": "*"
```

Comment:
this needs to be added to the turbo.json as a build dep as well

Reply:
good call!